modules/express/src/clientRoutes.ts

```diff
@@ -114,7 +114,6 @@ function handleSendCoins(req: express.Request) {
     .wallets()
     .get({ id: req.params.id })
     .then(function (wallet) {
-      // signs a tx, needs custom signing function passed in
```

Comment:
Are we not supporting external signer mode for these functions where the comments are getting removed?

Reply:
Yeah I believe that's the idea since they are all v1 wallet functions. Currently if signer mode is enabled, the v1 wallet functions would just operate the same as before.